packages/preview/sheen-id/0.1.0/typst.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "sheen-id"
```

Comment:
To use `sheen-id` as a package name we need to verify that you are affiliated with `sheen.id`. Could you send an email from your company email to hello@typst.app endorsing this package?